Libraries/Components/Pressable/Pressable.js

```diff
@@ -240,6 +244,25 @@ function Pressable(props: Props, forwardedRef): React.Node {
   _accessibilityState =
     disabled != null ? {..._accessibilityState, disabled} : _accessibilityState;
 
+  const accessibilityValue = {
+    max:
+      props['aria-valuemax'] !== null
+        ? props['aria-valuemax']
+        : props.accessibilityValue?.max,
```

Comment:
```suggestion
    max: props['aria-valuemax'] ?? props.accessibilityValue?.max,
```

This is more accurate and simple, please update the change for rest of properties on all changed files.